src/pycatcher/catch.py

```diff
@@ -651,5 +651,6 @@ def detect_outliers_stl(df) -> Union[pd.DataFrame, str]:
             print(return_outliers)
 
         logging.info("Completing outlier detection using STL")
+        return return_outliers
```

Comment:
Return statement is missing for some reason!

Reply:
@aseemanand could you please check and confirm if I got this right?

Reply:
We can remove print if return is used. It’ll re-print outliers if both are used. I can make the change.